frontend/src/app/libs/auth-pages/sign-in/sign-in.component.html

```diff
@@ -0,0 +1,32 @@
+<div>
+  <form [formGroup]="signInFormGroup" (ngSubmit)="submit()">
+    <div>
+      <mat-form-field appearance="fill">
+        <mat-label>Enter your email</mat-label>
+        <input matInput placeholder="pat@example.com" type="email" formControlName="email" />
+        <mat-error>{{ emailInputErrorMessage }}</mat-error>
+      </mat-form-field>
+    </div>
+
+    <div>
+      <mat-form-field appearance="fill">
+        <mat-label>Enter your password</mat-label>
+        <input matInput [type]="hidePassword ? 'password' : 'text'" formControlName="password" autocomplete="on" />
+        <mat-error>{{ passwordInputErrorMessage }}</mat-error>
```

Comment:
add ngIf condition to show mat-error or not (move from getter method, and make it only return error message)

Reply:
done